internal/querynode/query_node.go

```diff
@@ -226,13 +231,22 @@ func (node *QueryNode) Init() error {
 		node.etcdKV = etcdkv.NewEtcdKV(node.etcdCli, Params.EtcdCfg.MetaRootPath)
 		log.Info("queryNode try to connect etcd success", zap.Any("MetaRootPath", Params.EtcdCfg.MetaRootPath))
 
-		node.metaReplica = newCollectionReplica()
+		cpuNum := runtime.GOMAXPROCS(0)
+		node.cgoPool, err = concurrency.NewPool(cpuNum)
```

Comment:
this pool may be created with ants.WithPreAlloc(true)?

Reply:
Good suggestion!